lib/manifestTools/transformations/windows10.js

```diff
@@ -66,18 +68,45 @@ function convertFromBase(manifestInfo, callback) {
         if (isValidIconFormat(icon, validIconFormats)) {
           var iconDimensions = icon.sizes.split('x');
           if (iconDimensions[0] === '44' && iconDimensions[1] === '44') {
-            icons['44x44'] = { 'url': icon.src, 'fileName': 'smalllogo.scale-100.png' };
+            icons['44x44'] = { 'url': icon.src, 'fileName': c.WIN10_44_ICON_FILENAME };
+            maxResolutionIconSquare = (maxResolutionIconSquare.value <= 44) ? { value : 44 , src : icon.src } : maxResolutionIconSquare;
```

Comment:
I'll need to run this code to tell for sure, but is it backwards? it looks like it is pulling the 44x44 image from the w3c manifest, and then setting it to the maxResolutionIconSquare.  What I think we want is for us to look through the w3c manifest that is bigger than 44x44, and use it in the case that the 44x44 icon wasn't put in the manifest.  Maybe I am just missing the logic here, can you walk me through it?